src/include/migraphx/program.hpp

```diff
@@ -67,6 +69,8 @@ struct program
 
     void perf_report(std::ostream& os, std::size_t n, parameter_map params) const;
 
+    void roctx(const parameter_map& params, marker&& m);
```

Comment:
This should be called `mark` not `roctx`.

Reply:
Ah yes, as we used `mark` or `marker` for everything else. Makes sense.

For migraphx-driver command, I am keeping it `roctx` what do you think?